src/Peers/provider.cpp

```diff
@@ -78,10 +79,30 @@ void Provider::listen() {
             server->closeConn();
         }
 
-        if (task->getLeaderUuid() == uuid) {
-            leaderHandleTaskRequest(requesterIpAddr);
-        } else {
-            followerHandleTaskRequest();
+        try {
+            // Spin up python script child process
+            std::string script_path = "sorting_python/sorting.py";
+            std::vector<std::string> args{
+                std::to_string(zmq_sender.getPort()),
+                std::to_string(zmq_receiver.getPort())};
+            boost::process::child python_script("/usr/bin/env", "python3", script_path, args);
```

Comment:
is this dependent on where a user would have their local python installation? for me, when i run `where python3`, i see that my python installation is in `/usr/bin/python3`. that won't work with this setup then, right?